vault.py

```diff
@@ -580,6 +585,43 @@ def api_vault_collection_details(ctx, path):
 
     basename = pathutil.basename(path)
 
+    # If specific data package is selected, retreive all published versions
```

Comment:
Move this inside the "Check if collection is vault package" if else clause for a safer way to check this.